modules/apps/frontend-taglib/frontend-taglib-clay/src/main/java/com/liferay/frontend/taglib/clay/data/set/filter/BaseDateRangeClayDataSetFilter.java

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (c) 2000-present Liferay, Inc. All rights reserved.
+ *
+ * This library is free software; you can redistribute it and/or modify it under
+ * the terms of the GNU Lesser General Public License as published by the Free
+ * Software Foundation; either version 2.1 of the License, or (at your option)
+ * any later version.
+ *
+ * This library is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS
+ * FOR A PARTICULAR PURPOSE. See the GNU Lesser General Public License for more
+ * details.
+ */
+
+package com.liferay.frontend.taglib.clay.data.set.filter;
+
+/**
+ * @author Luca Pellizzon
+ */
+public abstract class BaseDateRangeClayDataSetFilter
+	implements ClayDataSetFilter {
+
+	public abstract DateClayDataSetFilterItem getMax();
+
+	public abstract DateClayDataSetFilterItem getMin();
+
+	public String getPlaceholder() {
+		return "dd/mm/yyyy";
```

Comment:
@luca-pellizzon this is wrong as it depends on the locale of the user

Reply:
it is not even used, I'm going to remove it
the placeholder is managed by the frontend component